test-cases/tests/installation/a34-verify-sku-values.md

```diff
@@ -0,0 +1,36 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+      - osd-post-upgrade
+    targets:
+      - 1.4.0
```

Comment:
```suggestion
      - 1.6.0
```
I am not sure if it needs to have "targets" set at all, but if yes, then 1.6.0 probably makes more sense :)

Reply:
@psturc @trepel would be interested in your thoughts here. I agree we should start from 1.6.0 but should we do it every install. The manual steps are themselves quite small as we've done work to automate the checks needed to a fairly thorough value. The automation is A34.